src/consensus/aft/raft_types.h

```diff
@@ -121,12 +134,25 @@ namespace aft
     Term term_of_idx;
     bool contains_new_view;
   };
+  DECLARE_JSON_TYPE_WITH_2BASES(
+    AppendEntries, RaftHeader, consensus::AppendEntriesIndex);
+  DECLARE_JSON_REQUIRED_FIELDS(
+    AppendEntries,
+    term,
+    prev_term,
+    leader_commit_idx,
+    term_of_idx,
+    contains_new_view);
```

Comment:
Also could you help me to understand what this field is used for?

Reply:
It seems that it introduced to gate a demand for evidence if a range contained a new view in the Byzantine case: https://github.com/microsoft/CCF/commit/55be5ab7811d38dcccce96875fcd9844e22c605a

It is, as far as I can tell, no longer used. We would need to make the deserialiser permissive if we removed it obviously, because code upgrades involve mixed networks.

Reply:
Moved to #5212